### PR TITLE
Grid overlay unit tests

### PR DESCRIPTION
**Pull request by Ea10**
Unit tests for the grid overlay feature as per issue #82

### CODE REVIEW

**Comment by nemotime** [comment]
No unnecessary imports, concise tests, good. I can't imagine any values that should be asserted in this case.

File: app/src/test/java/Test_GridOverlay.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    fun setUp() {
+        sactivity = Robolectric.setupActivity(SettingsActivity::class.java)
+    }
+
+    // test for checking if the toggle grid switch is there
+    @Test
+    fun test_ToggleGrid_Switch(){
+        val toggleGridHolder = sactivity!!.findViewById<RelativeLayout>(R.id.settings_turn_grid_on_holder)
+        val toggleGridSwitch = sactivity!!.findViewById<MySwitchCompat>(R.id.settings_turn_grid_on)
+        assertNotNull("Holder for 'toggle grid' is not present", toggleGridHolder)
```
Comment (StevenHaa) [inline app/src/test/java/Test_GridOverlay.kt]: is there a reason why you didn't say assertNull("Holder for 'toggle grid' is present", toggleGridHolder) instead?

Comment (Ea10) [reply app/src/test/java/Test_GridOverlay.kt]: The message is for when the assertion fails

Comment (StevenHaa) [reply app/src/test/java/Test_GridOverlay.kt]: ohhh i didn't know that, thanks that makes more sense. Good work!

**Pull request merged**